docs/guides/manage_keys.md

```diff
@@ -1,10 +1,11 @@
 # Manage Keys
+This document explains how to manage keys when using **Concrete**, introducing the key management API for generating, reusing, and securely handling keys.
 
-**Concrete** generates keys for you implicitly when they are needed and if they have not already been generated. This is useful for development, but it's not flexible **(or secure!)** for production. Explicit key management API is introduced to be used in such cases to easily generate and re-use keys.
+**Concrete** generates keys implicitly when needed. While this is convenient for development, it's not ideal for the production environment. The explicit key management API is available for you to easily generate and reuse keys as needed.
```

Comment:
`**Concrete** generates keys lazily when needed` ?

Reply:
will be done in https://github.com/zama-ai/concrete-internal/issues/840, let's merge this PR to not make Yuxi wait